cpp/util/etcd_delete.h

```diff
@@ -11,6 +11,10 @@
 namespace cert_trans {
 
 
+// Delete keys in batches (implemented using concurrent requests). The
+// "keys" argument are pairs of key and modified index. If the index
+// for an entry is zero, EtcdClient::ForceDelete will be used instead
+// of EtcdClient::Delete.
```

Comment:
One of the issues is that I was trying to reuse the internals and avoid changing them too much...

It's doable, but I'm pondering changing the signature altogether to be only "force", since that's the only spot we're using it. What do you think?